media/js/mozorg/home/home-a.js

```diff
@@ -97,38 +97,40 @@
 
     function enableWaypoints() {
         // fire the waypoints for each section, passing classes for the current and previous sections
-        introWaypoint = new Waypoint({
```

Comment:
If this is no longer being used, can `var introWaypoint` and `introWaypoint.destroy();` be removed too?